audit-ci.jsonc

```diff
@@ -1,32 +1,6 @@
 {
 	// $schema provides code completion hints to IDEs.
 	"$schema": "https://github.com/IBM/audit-ci/raw/main/docs/schema.json",
-	"low": true,
-	"allowlist": [
-
-		// The package "taffydb" is only used during jsdoc build and contains only data for jsdoc generation.
-		// After jsdoc build finished, db is destroyed.
-		// According to JSDoc, this should not be considered a security risk:
-		// https://github.com/jsdoc/jsdoc/tree/main/packages/jsdoc-salty#taffydb-jsdoc-and-security
-		"GHSA-mxhp-79qh-mcx6",
```

Comment:
Isn't this one "high"?

Reply:
Thanks! I ran it locally on with v3 dependencies and it made me think wrongly about it as lower vulnerability. Now should be ok